src/components/application_manager/test/commands/hmi/get_urls_test.cc

```diff
@@ -97,12 +98,34 @@ class GetUrlsTest : public CommandsTest<CommandsTestMocks::kIsNice> {
         utils::MakeShared<policy_manager_test::MockPolicyManager>();
     ASSERT_TRUE(mock_policy_manager_);
   }
+
+  MessageSharedPtr CreateCommandMsg() {
+    MessageSharedPtr command_msg(
+        CreateMessage(NsSmartDeviceLink::NsSmartObjects::SmartType_Map));
+    (*command_msg)[am::strings::params][am::strings::connection_key] =
+        kConnectionKey;
+    (*command_msg)[am::strings::msg_params][am::hmi_request::service] =
+        kPolicyService;
+
+    return command_msg;
+  }
+
+  RequestFromHMIPtr PrepareRunFlowAndCreateCommandRequestFromHMI(
+      MessageSharedPtr command_msg) {
+    ON_CALL(mock_app_manager_, event_dispatcher())
+        .WillByDefault(ReturnRef(mock_event_dispatcher_));
+
+    RequestFromHMIPtr command(CreateCommand<GetUrls>(command_msg));
+
+    EXPECT_CALL(mock_app_manager_, GetPolicyHandler()).Times(3);
```

Comment:
@dcherniev, correctness of command execution is not depends on `GetPolicyHandler()` call, so we can use `ON_CALL`. And if you do as I mentioned in the previous comment, it can be also mover to constructor.